pkg/autoscaler/multiscaler.go

```diff
@@ -187,7 +187,8 @@ func (m *MultiScaler) Create(ctx context.Context, decider *Decider) (*Decider, e
 	}
 	scaler.mux.RLock()
 	defer scaler.mux.RUnlock()
-	return (&scaler.decider).DeepCopy(), nil
+	// scaler.decider is already a copy of the original, so just return it.
+	return scaler.decider, nil
```

Comment:
it's a little weird that we're returning the thing we are passed, though I don't understand why a copy was returned previously when that's not what was stored internally.

Reply:
It is to mimick the API server behavior I guess.

Reply:
We store immutable copy in the uniscaler and scalerunner.
While the mutable copy is inside multiscaler, so multiscaler implements "API server" functionality and hence returns a copy. Perhaps overthought, but oh well.